.github/workflows/build.yml

```diff
@@ -452,7 +458,16 @@ jobs:
         # because examples enable rustls' features, `--workspace --no-default-features` is not
         # the same as `--package rustls --no-default-features` so run it separately
       - run: cargo clippy --locked --package rustls --no-default-features --all-targets -- $CLIPPY_PARAMS
-      - run: cargo clippy --locked --workspace --all-features --all-targets -- $CLIPPY_PARAMS
+      - run: |
+          set -x
+          for p in $(admin/all-workspace-members) ; do
+            if [ "$p" == "bogo" ] ; then
+              ALL_FEATURES="--features fips"
+            else
+              ALL_FEATURES="--all-features"
+            fi
+            cargo clippy --locked --package $p $ALL_FEATURES --all-targets -- $CLIPPY_PARAMS
+          done
```

Comment:
At the point where we're writing Python anyway WDYT about expanding the scope of `all-workspace-members` to take these details as input, doing more of the looping/executing a cmd itself?

The shell scripting here isn't too egregious but it's also a pretty gross thing to have to use (does it need `set -e`?) and being embedded in YAML is like two terrible tastes that taste terrible together :laughing:

Reply:
> I expect it to be temporary and hopefully fixed in the next aws-lc-rs release. Then the second commit can be reverted.

Sorry, wasn't thinking about this comment. Given this knowledge I think it's fine as-is.

Reply:
It's a reasonable point -- I have hoisted all the clippy stuff into one script; this also reduces some of the repetition in the yaml.